src/api/routineLikeApi.ts

```diff
@@ -0,0 +1,14 @@
+import api from '@/config/axios/config';
+import { HttpStatusCode } from 'axios';
+
+export enum RoutineLike_Endpoint {
+  ROUTINE_LIKE = 'api/workout/routine/like',
+}
+
+export const like = async (id: string, method: 'POST' | 'DELETE'): Promise<HttpStatusCode> => {
```

Comment:
it shouldnt be like this, it creates some inconvinience.
Better to create to methods